doc/Type/UInt.pod6

```diff
@@ -0,0 +1,12 @@
+=begin pod
+
+=TITLE class UInt
```

Comment:
On Wed, Aug 31, 2016 at 10:11 AM, Zoffix Znet <notifications@github.com> wrote:
> In doc/Type/UInt.pod6:
>
> > @@ -0,0 +1,12 @@
> > +=begin pod
> > +
> > +=TITLE class UInt
>
> This probably should be subset not class. You can't instantiate a subset, for example, but you can a class.

Hm, on #perl6 I did:

  m: my UInt $u; ...

and timotimo did, too, with no problems.